installing/installing_bare_metal_ipi/ipi-install-overview.adoc

```diff
@@ -4,19 +4,24 @@
 include::_attributes/common-attributes.adoc[]
 :context: ipi-install
 
-Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installation.
-
-During installer-provisioned installation on bare metal, the installation program on the bare metal node labeled as `provisioner` creates a bootstrap virtual machine (VM). The role of the bootstrap VM is to assist in the process of deploying an {product-title} cluster. The bootstrap VM connects to the `baremetal` network and to the `provisioning` network, if present, via the network bridges.
+Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installer-provisioned base-metal installation. The following diagram illustrates the installation environment in phase one of deployment.
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_1.png[Deployment phase one]
 
-The provisioning node can be removed after the installation.
 
-When the installation of {product-title} is complete and fully operational, the installation program destroys the bootstrap VM automatically and moves the virtual IP addresses (VIPs) to the appropriate nodes accordingly. The API VIP moves to the control plane nodes and the Ingress VIP moves to the worker nodes.
+- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the controller of a new openshift cluster.
+- **Bootstrap VM**: A virtual machine that assists in the process of deploying an {product-title} cluster.
+- **Network bridges**: The bootstrap VM connects to the baremetal network and to the provisioning network, if present, via network bridges, `eno1` and `eno2`.
+
+In phase 2 of the deployment, the provisioner destroys the bootstrap VM automatically and moves the virtual IP addresses (VIPs) to the appropriate nodes accordingly. The API VIP moves to the control plane nodes and the Ingress VIP moves to the worker nodes.
```

Comment:
Paragraph above mentions "phase one", whereas here it's "phase 2"

Per the ISG: https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words
"In general, use numerals to represent all numbers, however large or small. For numbers below 10, it is alternatively acceptable to use words instead of numerals. Within a publication, be consistent in your approach."

I think it's OK to use either words or numerals, but we should stick with one consistently.

Reply:
Standardised on numerals.